docs/import/add_disaster.js

```diff
@@ -191,7 +237,17 @@ function withText(td, layer, property) {
  * @return {JQuery<HTMLTableDataCellElement>}
  */
 function withList(td, layer, property) {
-  return td.text(layer[property][0]);
+  const list = layer[property];
+  let withNewLines = list[0];
+  for (let i = 1; i < list.length; i++) {
+    withNewLines = withNewLines.concat('\n', list[i]);
+  }
+  // use a regex to replace all instances of ',' with a new line
+  const textarea =
+      $(document.createElement('textarea')).val(withNewLines).prop('rows', 3);
+  td.append(textarea);
+  td.on('blur', 'textarea', (event) => onListBlur(event, property));
```

Comment:
why not trigger on the textarea directly, rather than the td? Here and above with the input.

Reply:
Sure